### PR TITLE
update text to reflect jQuery 3 update

### LINKED ISSUE #4453: update jQuery's setTimeout reference in "Optimize JavaScript Execution"
This line
```
In fact, jQuery’s default animate behavior today is to use setTimeout! 
```
in [Optimize JavaScript Execution](https://developers.google.com/web/fundamentals/performance/rendering/optimize-javascript-execution) is no longer true as of [jQeury 3.0](https://jquery.com/upgrade-guide/3.0/#feature-animations-now-use-requestanimationframe)  🙌

cc: @paullewis

### PR DESCRIPTION
**Pull request by kosamari**
closes #4453

### CODE REVIEW

File: src/content/en/fundamentals/performance/rendering/optimize-javascript-execution.md [hunk 1]
```diff
@@ -48,7 +48,9 @@ Frameworks or samples may use `setTimeout` or `setInterval` to do visual changes
 
 <img src="images/optimize-javascript-execution/settimeout.jpg" alt="setTimeout causing the browser to miss a frame.">
 
-In fact, jQuery’s default `animate` behavior today is to use `setTimeout`! You can [patch it to use `requestAnimationFrame`](https://github.com/gnarf/jquery-requestAnimationFrame), which is strongly advised.
+In fact, jQuery used to use setTimeout for animate behavior. It was changed to use requestAnimationFrame in version 3.
```
Comment (PaulKinlan) [inline src/content/en/fundamentals/performance/rendering/optimize-javascript-execution.md]: `requestAnimationFrame` backticks around requestAnimationFrame

Comment (mathiasbynens) [inline src/content/en/fundamentals/performance/rendering/optimize-javascript-execution.md]: And around `setTimeout` and `animate`, too. Something like:

> In fact, jQuery used to use `setTimeout` for its `animate` behavior. It was changed to use `requestAnimationFrame` in version 3.

Comment (kosamari) [inline src/content/en/fundamentals/performance/rendering/optimize-javascript-execution.md]: eeeeeee thanks for catching that, that was brain farting after 4 hours on video shoot. Fixed now.

File: src/content/en/fundamentals/performance/rendering/optimize-javascript-execution.md [hunk 2]
```diff
@@ -44,19 +53,34 @@ When visual changes are happening on screen you want to do your work at the righ
...
+You should be tactical about when JavaScript runs, and for how long. For example, if you’re in an 
+animation like scrolling, you should ideally be looking to keep your JavaScript to something in the 
+region of **3-4ms**. Any longer than that and you risk taking up too much time. If you’re in an idle
+ period, you can afford to be more relaxed about the time taken.
 
-In many cases you can move pure computational work to [Web Workers](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/basic_usage), if, for example, the it doesn’t require DOM access. Data manipulation or traversal, like sorting or searching, are often good fits for this model, as are loading and model generation.
+In many cases you can move pure computational work to 
+[Web Workers](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/basic_usage), 
+if, for example, the it doesn’t require DOM access. Data manipulation or traversal,
```
Comment (jpmedley) [inline src/content/en/fundamentals/performance/rendering/optimize-javascript-execution.md]: Should "the it doesn't" be "it doesn't"?

**Comment by jpmedley** [comment]
Regarding the warning about wf_blink_components, you can find the [complete list here](https://github.com/google/WebFundamentals/blob/master/src/data/blinkComponents.json). If you're not sure that the blink component is relevant, you can ignore the warning. I'd rather have none listed than the wrong one listed.

**Pull request merged**